src/services/recipient.test.js

```diff
@@ -164,8 +164,8 @@ describe('Recipient DB service', () => {
         programType: 'EHS',
         startYear: 'Aeons ago',
         status: 'active',
-        startDate: 'today',
```

Comment:
Was there any case you found were this was expecting a string value like 'today'? Or is this just bad test data?